setup.py

```diff
@@ -28,7 +28,8 @@
     packages=[
         'djangocms_snippet',
         'djangocms_snippet.migrations',
-        'djangocms_snippet.south_migrations'
+        'djangocms_snippet.south_migrations',
+        'djangocms_snippet.templatetags'
```

Comment:
If you add a trailing `,` at the end of the line, at the next change it will not appear in the diff, which is much nicer.